packages/protect/src/ffi/operations/decrypt.ts

```diff
@@ -42,18 +42,23 @@ export class DecryptOperation
           throw noClientError()
         }
 
-        if (this.encryptedData === null) {
+        if (this.encryptedData?.data === null) {
           return null
         }
 
-        if (this.encryptedData.k !== 'ct') {
+        if (this.encryptedData.data.k !== 'ct') {
```

Comment:
The `kind` is no longer required

Reply:
Does that mean that we need to remove the kind [on the FFI side as well](https://github.com/cipherstash/protectjs-ffi/blob/4710c8f974ccc07415361894be45bb7592c5c732/crates/protect-ffi/src/lib.rs#L42) @tobyhede?

For customers migrating to the new version, will the DB migration involve wrapping the value in the composite type and removing the kind field?

Reply:
Yes, can be removed from FFI.
Is unused now, so doesn't **have** to be migrated. Just extra data that will be removed when records are updated over time. 
Kind was required because in the original MLP we were sending the plaintext as a json payload. 
Protect.js handles the encryption now, and Proxy does the mapping so the storage format is the only variant we need.

Reply:
Will clean up in a follow up PR since the EqlSchema relies on the k key to determine the type